src/Providers/ScoutApmServiceProvider.php

```diff
@@ -148,6 +164,17 @@ public function boot(
         $this->instrumentMiddleware($httpKernel);
     }
 
+    private function shouldInstrumentQueues(Config $config) : bool
+    {
+        $configValue = $config->get(self::CONFIG_INSTRUMENT_LARAVEL_QUEUES);
+
+        if ($configValue === null) {
```

Comment:
Note: depending on the root cause of the issue #43, it may be sensible to default the queue instrumentation to `false` not `true` if not explicitly configured.